src/main/java/com/parkhomovsky/bookstore/controller/OrderController.java

```diff
@@ -0,0 +1,76 @@
+package com.parkhomovsky.bookstore.controller;
+
+import com.parkhomovsky.bookstore.dto.order.OrderDto;
+import com.parkhomovsky.bookstore.dto.order.OrderPlaceRequestDto;
+import com.parkhomovsky.bookstore.dto.order.OrderUpdateStatusRequest;
+import com.parkhomovsky.bookstore.dto.order.UpdateResponseDto;
+import com.parkhomovsky.bookstore.dto.orderitem.OrderItemDto;
+import com.parkhomovsky.bookstore.exception.EmptyShoppingCartException;
+import com.parkhomovsky.bookstore.service.OrderService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.Authentication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/orders")
+@Tag(name = "Orders management",
+        description = "Endpoints for managing user orders")
+public class OrderController {
+    private final OrderService orderService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(summary = "Place new order",
+            description = "Create a new order and add it to the database")
+    public OrderDto placeOrder(@RequestBody OrderPlaceRequestDto orderPlaceRequestDto)
+            throws EmptyShoppingCartException {
```

Comment:
If you handle it in the global handler you don't need to make it checked exception, make it extend RuntimeException and remove `throws`